chemberta/masked-lm/train_roberta_regression.py

```diff
@@ -0,0 +1,124 @@
+""" Script for training a Roberta Regression Model (single or multi-task)
```

Comment:
There's a lot of code overlap between `train_roberta_regression.py` and `train_roberta_sequenceclass.py` - would you consider refactoring these into a single script?

Reply:
I believe we discussed combining both in Slack earlier once normalizing + overfit tests worked, so we should be able to do so! I can take a crack at this if @ElanaPearl is ok with merging this in.